app-pda/libplist/libplist-1.12.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="A library to handle Apple Property List format in binary or XML"
+DESCRIPTION="libplist is a library for reading and writing \
+the Apple binary and XML property list format. It is part of \
+the libimobiledevice stack providing access to iDevices."
+HOMEPAGE="http://www.libimobiledevice.org"
+COPYRIGHT="2009 Nikias Bassen"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://www.libimobiledevice.org/downloads/$portName-$portVersion.tar.bz2"
+CHECKSUM_SHA256="0effdedcb3de128c4930d8c03a3854c74c426c16728b8ab5f0a5b6bdc0b644be"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	libplist$secondaryArchSuffix = $portVersion compat >= 0
+	cmd:plistutil$secondaryArchSuffix = $portVersion compat >= 0
+	lib:libplist$secondaryArchSuffix = $portVersion compat >= 0
+	lib:libplist++$secondaryArchSuffix = $portVersion compat >= 0
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libglib2.0$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libplist${secondaryArchSuffix}_devel = $portVersion compat >= 0
+	devel:libplist$secondaryArchSuffix = $portVersion compat >= 0
+	devel:libplist++$secondaryArchSuffix = $portVersion compat >= 0
+	"
+REQUIRES_devel="
+	libplist$secondaryArchSuffix == $portVersion base
+	devel:libxml2$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libxml2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:cmake
+	cmd:python
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:sh
+	"
+BUILD()
+{
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	# remove libtool file
```

Comment:
Cool, we're reaching perfection!
The only remaining changes that I see are:
* changing line 65 to:
```
	# remove libtool files
```

* checking if **`devel:libglib_2.0$secondaryArchSuffix`** should be added to **`REQUIRES_devel`**.

To see if **`devel:libglib_2.0`** should be added to **`REQUIRES_devel`** one needs to browse the header files that get packaged in **`libplist_devel`** and see if any of them refer to any header in the **`glib2_devel`** package.

That said, If you're not sure whether **`libplist_devel`** needs **`libglib2_devel`** then I think you can pretend it does (and, consequently, add **`devel:libglib_2.0$secondaryArchSuffix`**  to **`REQUIRES_devel`**) because it doesn't hurt ;-)

To all: if anyone knows that **`libplist_devel`** does not refer to any header files in **`libglib2_devel`** please let us know cause I'm not running Haiku at the moment so I can't check it right now. Thanks!